fab/fabfile.py

```diff
@@ -332,8 +332,8 @@ def hotfix_deploy():
     run('echo ping!')  # workaround for delayed console response
     try:
         execute(release.update_code, env.deploy_metadata.deploy_ref, True)
-    except Exception:
-        execute(mail_admins, "Deploy failed", "You had better check the logs.")
+    except Exception, e:
+        execute(mail_admins, "Deploy failed", u"Exception message:\n{exc}".format(exc=e))
```

Comment:
I thought this traceback would be way too long for an email, but I will add it and revisit if it's too long latter.